hack/run-ci-e2e-test.sh

```diff
@@ -136,7 +136,8 @@ echo "Testing against Windows Server $WIN_VER"
 # The bool flags in golang does not respect key value pattern. They follow -flag=x pattern.
 # -flag x is allowed for non-boolean flags only(https://golang.org/pkg/flag/)
 
-GO_TEST_ARGS="$BYOH_NODE_COUNT_OPTION $MACHINE_NODE_COUNT_OPTION --private-key-path=$KUBE_SSH_KEY_PATH --wmco-namespace=$WMCO_DEPLOY_NAMESPACE --windows-server-version=$WIN_VER"
+GO_TEST_ARGS="$BYOH_NODE_COUNT_OPTION $MACHINE_NODE_COUNT_OPTION --private-key-path=$KUBE_SSH_KEY_PATH --wmco-namespace=$WMCO_DEPLOY_NAMESPACE --windows-server-version=$WIN_VER --mirror-registry=$MIRROR_REGISTRY_HOST"
+echo $GO_TEST_ARGS
```

Comment:
Please remove or add more context to the print statement

Reply:
I lean towards keeping the log but adding more context

Reply:
by context I meant what is actually being printed out, not a comment, it is clear via the code and a comment in not needed.

For example
```
echo "Running tests with arguments: $GO_TEST_ARGS"
```